CHANGELOG.md

```diff
@@ -4,9 +4,37 @@ See the [change log guidelines](http://keepachangelog.com/) for information on h
 
 ## [Unreleased]
 
+## [3.2.0]
+
 ### Added
 
-- Added support for several Arc-based GML geometry types to be interpolated when read: Arc, ArcString, ArcByCenterPoint, Circle, CircleByCenterPoint. The interpolation is based on a Grid assuring a given maximum positional error.
+- Added support for several Arc-based GML geometry types to be interpolated when read: Arc, ArcString, Circle, CircleByCenterPoint. The interpolation is based on one of two algorithms that can be selected on import.
+- Project Validator that validates exported instances based on validator configuration (e.g. rules or schemas) imported into the project
+- New transformation function `Assign collected values` allows the assignment of all values collected by a Groovy transformation function. The new function automatically converts collected values to references if the target property takes references.
+- Better usage of available space in Alignment and Mapping views
+- Content assistance for project variables in several UI wizards
+- Request pagination for WFS requests. Users can now choose to activate request pagination for WFS sources.
+- IO Provider extensions can now have a configurationContentType to describe the content type of configuration files for this provider
+- Total number of imported instances is now shown in progress dialog (if known)
+- InstanceResolver now allows resolving multiple references at once
+
+### Changed
+
+- `Validator rules` are now called `Validator configurations`
+- The Merge function now uses an iterative approach for merging instances which allows for processing more data in a Merge
+- RestrictedGroovyInterceptor now allows SimpleDateFormat and UUID classes
+- When loading data from CSV files the data is now streamed (similar to XML data sources) and not loaded at once into memory
+
+### Fixed
+
+- Prevent multiple message boxes during validation when multiple validators are executed
```

Comment:
Agreed, I was trying to keep things out that were introduced in this release. This one slipped through.